packages/twenty-front/src/modules/workspace/components/WorkspaceInviteTeam.tsx

```diff
@@ -79,16 +79,30 @@ export const WorkspaceInviteTeam = () => {
     },
   });
 
-  const submit = handleSubmit(async (data) => {
-    const emailsList = sanitizeEmailList(data.emails.split(','));
-    const result = await sendInviteLink({ variables: { emails: emailsList } });
-    if (isDefined(result.errors)) {
-      throw result.errors;
+  const submit = handleSubmit(async ({ emails }) => {
+    const emailsList = sanitizeEmailList(emails.split(','));
+    const { data } = await sendInvitation({ emails: emailsList });
+    if (isDefined(data)) {
+      if (data.sendInviteLink.result.length > 0) {
+        enqueueSnackBar(
+          `${data.sendInviteLink.result.length} invitations sent`,
+          {
+            variant: SnackBarVariant.Success,
+            duration: 2000,
+          },
+        );
+      }
+      if (!data.sendInviteLink.success) {
+        data.sendInviteLink.errors.forEach((error) => {
+          enqueueSnackBar(error, {
+            variant: SnackBarVariant.Error,
+            duration: 5000,
+          });
+        });
+      }
+    } else {
```

Comment:
I don't think the else is needed here!

However, we usually prefer the following pattern:
```
if (edgeCase) { return }
if (edgeCase} { return }
myNominalBehavior()
```

over
```
if (!edgeCase) { 
if (!edgeCase) {
 myNominalBehavior() 
}}
```
as this avoid nested if statements which is easier for the brain to understand as edge cases are eliminated sooner and sequentially